amnestywebsite/humanity-theme
# Replace archive.php with FSE templates

Ref: https://github.com/amnestywebsite/humanity-theme/issues/369

**Steps to test**:
1. checkout and run an asset build
2. for each language, visit both /latest/ and /latest/news/
3. each should render as normal, and still function correctly
